site/pages/create-social-media-post/[disruptionId]/[socialMediaPostIndex].page.tsx

```diff
@@ -46,6 +58,70 @@ const CreateSocialMediaPost = (props: CreateSocialMediaPostPageProps): ReactElem
         (account) => account.id === pageState.inputs.socialAccount,
     )?.accountType;
 
+    const removeGroupId = (e: SyntheticEvent, index: number) => {
+        e.preventDefault();
+        if (pageState.inputs.accountType === "Nextdoor" && pageState.inputs.groupIds) {
+            const groupIds = [...pageState.inputs.groupIds];
+            groupIds.splice(index, 1);
+            setPageState({
+                ...pageState,
+                inputs: {
+                    ...pageState.inputs,
+                    groupIds,
+                },
+            });
+        }
+    };
+
+    const getGroupIdRows = () => {
```

Comment:
I'm not sure groupId is the value we want to show as it might not mean much to the user.

I think the title of the input should be "Custom Area" and when we call the get agency boundary API we should bring back the name of the Custom Area and put that in the dropdown rather than the ID.

(This is a bit of a guess from logging into our test nextdoor account and clicking around on it. Maybe let's jump on a call together and double check this - also might be better to wait until the backend is wired up to make this change as it'll make more sense then)

Reply:
done in https://github.com/Department-for-Transport-Disruptions/create-disruptions-data/pull/418